graphql/schema.js

```diff
@@ -150,7 +150,10 @@ type Connection {
   from: Entity!
   to: Entity!
   vias: [Connection!]!
+  # Last available compensation
   compensation: Compensation
+  # Yearly compensations
+  compensations: [Compensation!]
```

Comment:
```suggestion
  compensations: [Compensation!]!
```
This means the API will always deliver an array, it might be empty or filled with valid Compensation objects (null or invalid objects that get nulled not allowed). Otherwise you would need to handle `compensations` being falsey in [`components/Connections`](https://github.com/lobbywatch/website/pull/18/files#diff-2464df2cc320d3522dd4689294f5ca9dR364).

Make sure to test a case without compensations data.

Reply:
> This means the API will always deliver an array, it might be empty or filled with valid Compensation objects (null or invalid objects that get nulled not allowed).

Yeah, exactly.

I've implemented the logic mainly in the dataIF.

> Make sure to test a case without compensations data.

![image](https://user-images.githubusercontent.com/876593/64909015-08916500-d707-11e9-9f85-9d05f405bb74.png)